deploy/001-source-timelocks.ts

```diff
@@ -18,60 +19,32 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     return live ? governorBravoDelegatorAddress : deployer;
   };
 
-  const delayConfig = {
-    hardhat: {
-      normal: 600,
-      fast: 300,
-      critical: 100,
-    },
-    bscmainnet: {
-      normal: 172800,
-      fast: 21600,
-      critical: 3600,
-    },
-    bsctestnet: {
-      normal: 600,
-      fast: 300,
-      critical: 100,
-    },
-    sepolia: {
-      normal: 600,
-      fast: 300,
-      critical: 100,
-    },
-    ethereum: {
-      normal: 172800,
-      fast: 21600,
-      critical: 3600,
-    },
-  };
-
   await deploy("NormalTimelock", {
     contract: "Timelock",
     from: deployer,
-    args: [getAdmin(), delayConfig[networkName].normal],
+    args: [getAdmin(), (delayConfig as any)[networkName].normal],
     log: true,
     autoMine: true,
   });
 
   await deploy("FastTrackTimelock", {
     contract: "Timelock",
     from: deployer,
-    args: [getAdmin(), delayConfig[networkName].fast],
+    args: [getAdmin(), (delayConfig as any)[networkName].fast],
     log: true,
     autoMine: true,
   });
 
   await deploy("CriticalTimelock", {
     contract: "Timelock",
     from: deployer,
-    args: [getAdmin(), delayConfig[networkName].critical],
+    args: [getAdmin(), (delayConfig as any)[networkName].critical],
     log: true,
     autoMine: true,
   });
 };
 
-func.tags = ["Local", "Timelock"];
+func.tags = ["Local", "Timelock", "LocalTimelock"];
```

Comment:
How are these tags used? "Local"  and "LocalTimelock" only appear here "Timelock" appears here and in 005-remote-timelocks? so if you can run scripts based on more than one tag "Local" and "timelock" should be sufficient. Maybe we need clearer definitions on how tags are used?

Reply:
https://github.com/VenusProtocol/governance-contracts/pull/57/commits/130bac81b04c0b3996bc7c16d2e55c39e9b5c941